src/erllambda_poller.erl

```diff
@@ -115,7 +115,7 @@ handle_info(poll, #state{handler = Handler} = State) ->
     {ok, ReqId, Hdrs, Body} = invoke_next(State),
     %% container thaw happens here
     erllambda:message("Next returns, in invoke ~p", [os:system_time(millisecond)]),
-    case erllambda:invoke(Handler, Body, Hdrs) of
+    case erllambda:invoke(Handler, Body, maps:to_list(Hdrs)) of
```

Comment:
`Hdrs` is list here and all map conversions deliberately to happen in handle process so not to pile up small binaries in poller between invokes